ami/adjust-cgroup-settings.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Exclude containers we don't want effected by limits
+EXCLUDES_PATTERN=$(cat <<'EOF' | xargs | sed 's/ /|/g'
+amazon/amazon-ecs-agent
+goodeggs/logspout-goodeggs
+yelp/docker-custodian
+convox/api
+convox/agent
+goodeggs/ranch-api
+EOF
+)
+
+# Build a list of stuff we DO want to effect
+TARGETS=$(docker ps --no-trunc | grep -v 'CONTAINER' | grep -Ev "$EXCLUDES_PATTERN" | awk '{ print $1; }' | xargs)
+
+# Apply Modifications
+for a in $TARGETS; do
+
+  # Disable swap (set memory and swap to same size)
+  pushd /cgroup/memory/docker/${a}
+  cat "./memory.limit_in_bytes" > "./memory.memsw.limit_in_bytes"
```

Comment:
so we're not limiting swap on the excluded containers.  seems ok but calling it out.